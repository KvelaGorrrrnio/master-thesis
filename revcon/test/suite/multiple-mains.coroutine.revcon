@coroutine

coroutine main() {
  a += 1
}

coroutine main() {
  a += 2
}
