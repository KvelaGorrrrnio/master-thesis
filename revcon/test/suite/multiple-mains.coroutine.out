Multiple coroutines named main was found.
Only one main must be defined.
